packages/pn-personafisica-webapp/src/components/Contacts/EmailSmsContactWizard.tsx

```diff
@@ -148,17 +151,47 @@ const EmailSmsContactWizard: React.FC = () => {
       .catch(() => {});
   };
 
+  const handleTrackValidationEvents = (
+    value: string,
+    channelType: ChannelType,
+    errors?: string
+  ) => {
+    // keep this log for the moment to avoid unusued variable error
+    // will be useful for sms event tracking since this function will be re-used
+    console.log(channelType);
+
+    if (!value) {
+      PFEventStrategyFactory.triggerEvent(PFEventsType.SEND_ADD_SERCQ_SEND_EMAIL_MISSING);
```

Comment:
For PEC we log a similar event but in that case, we send the event also if the pec is invalid. Here we are sending the error only when email is empty